### DISCUSSION TITLE
importing libraries seems to be failing

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#318] [General]
so I'm trying to use this extension https://github.com/Astropulse/sd-palettize with vladmandic's automatic and it seems to pop up this error:

![dasd](https://user-images.githubusercontent.com/120217469/233479096-d686ebea-5536-4127-b3bc-ff9346a14491.png)

every other extension works fine and this extension works fine with automatic1111. It seems that the error is provoked because the hitherdither library is not being installed automatically

**Comment by vladmandic** [comment]
not sure why create both discussion and issue for the same thing, resolved under #321

**Reply by ghost** [reply]
yeah my bad mate, first time using this github system
